template/typescript/base/src/components/HelloWorld.vue

```diff
@@ -1,32 +1,25 @@
 <template>
-  <v-container class="fill-height">
-    <v-responsive
-      class="align-centerfill-height mx-auto"
-      max-width="900"
-    >
+  <v-container class="fill-height" width="900">
```

Comment:
Should make this **max-width** so it scales on mobile.

Reply:
I had seen how it behaves below 900px - not pretty - and thought it was not intentional, but I don't really have a strong opinion here.